docs/source/api/osversions.rst

```diff
@@ -66,17 +66,17 @@ The **directory** of the file can be changed by creating a specific :term:`Param
 The format of the file is a JSON object as described in :ref:`response-structure`.
 
 .. code-block:: json
-        :caption: Example osversions.json file
+	:caption: Example osversions.json file
 
-        {
-          "CentOS 7.2": "centos72"
-        }
+	{
+			"CentOS 7.2": "centos72"
```

Comment:
Looks like this got indented one too many times.

Reply:
Adjusted in 0220c7acba.